Tools/CISupport/build-webkit-org/config.json

```diff
@@ -33,22 +33,22 @@
                   { "name": "bot258", "platform": "mac-ventura" },
                   { "name": "bot260", "platform": "mac-ventura" },
 
-                  { "name": "bot105", "platform": "mac-monterey" },
-                  { "name": "bot111", "platform": "mac-monterey" },
-                  { "name": "bot135", "platform": "mac-monterey" },
-                  { "name": "bot138", "platform": "mac-monterey" },
-                  { "name": "bot139", "platform": "mac-monterey" },
-                  { "name": "bot141", "platform": "mac-monterey" },
-                  { "name": "bot185", "platform": "mac-monterey" },
-                  { "name": "bot187", "platform": "mac-monterey" },
-                  { "name": "bot198", "platform": "mac-monterey" },
-                  { "name": "bot279", "platform": "mac-monterey" },
-                  { "name": "bot603", "platform": "mac-monterey" },
-                  { "name": "bot632", "platform": "mac-monterey" },
-                  { "name": "bot1021", "platform": "mac-monterey" },
-                  { "name": "bot1023", "platform": "mac-monterey" },
-                  { "name": "bot1025", "platform": "mac-monterey" },
-                  { "name": "bot1026", "platform": "mac-monterey" },
+                  { "name": "bot105", "platform": "*" },
```

Comment:
This should be platform: mac-ventura and be moved to the section with the other Ventura bots.